### PR TITLE
Updated warper to version 1.4b (some part of it perhaps related to th…

### PR DESCRIPTION
**Pull request by EPuncker**
…e issue 8666 as well)

### CODE REVIEW

File: npc/custom/warper.txt [hunk 1]
```diff
@@ -134,23 +141,23 @@ T9: Restrict("RE");
 T10: Go("einbech",63,35);
 T11: Go("einbroch",64,200);
 T12: Restrict("RE");
-	Go("dicastes01",198,187);
+	 Go("dicastes01",198,187);
 T13: Go("geffen",119,59);
 T14: Go("gonryun",160,120);
 T15: Go("hugel",96,145);
-T16: Go("izlude",128,114);
+T16: Go("izlude",128,(RENEWAL_EXP?146:114));
```
Comment (MishimaHaruna) [inline npc/custom/warper.txt]: This would be `RENEWAL`, not `RENEWAL_EXP`

File: npc/custom/warper.txt [hunk 2]
```diff
@@ -67,41 +75,40 @@ function Disp {
 		set @menu$, getarg(0);
 	else {
 		set @menu$,"";
-		for(set .@i,getarg(1); .@i<=getarg(2); set .@i,.@i+1)
-			set @menu$, @menu$+getarg(0)+" "+.@i+":";
+		for (.@i = getarg(1); .@i <= getarg(2); set .@i++)
```
Comment (dastgir) [inline npc/custom/warper.txt]: Only
 .@i++
Instead of 
set .@i++

Comment (EPuncker) [inline npc/custom/warper.txt]: I should not do things sleepy lol

**Comment by EPuncker** [comment]
I guess it is fine now

**Comment by dastgir** [comment]
Can you squash both commit into 1?

**Comment by EPuncker** [comment]
done

**Comment by dastgir** [comment]
Thanks for contribution

**Pull request merged**